teamclass.py

```diff
@@ -8,7 +8,7 @@
 import re
 
 
-regex = re.compile(r"^[a-zA-Z0-9 ]*$")
+regex = re.compile(r"[^a-zA-Z\d :.,'%`]")
```

Comment:
Updated the regex code to work properly, this works in 3.8 and now includes the following characters: :.,'%`